### PR TITLE
Gives several planet turfs names

### PR DESCRIPTION
**Pull request by Citinited**
This PR gives our turfs different names instead of "floor". Open turfs have their gender set to PLURAL, so that when examined, it says `That's some grass` instead of `That's a grass`.

It's only a small addition but improves muh immersions.

:cl:
tweak: Planet turfs are now more appropriately named.
/:cl:

### CODE REVIEW

File: code/game/turfs/simulated/planet.dm [hunk 1]
```diff
@@ -27,21 +30,25 @@
 		return
 
 /turf/open/floor/planet/dirt
+	name = "dirt"
 
 /turf/open/floor/planet/greenerdirt
 	icon_state = "greenerdirt"
 
 /turf/open/floor/planet/wood
+	name = "wooden floor"
```
Comment (Birdtalon) [inline code/game/turfs/simulated/planet.dm]: Won't this say "that's some wooden floor"?

Comment (Citinited) [inline code/game/turfs/simulated/planet.dm]: Oh wait, this should be `GENDER = NEUTER`.

Comment (Citinited) [inline code/game/turfs/simulated/planet.dm]: Yep, I spotted this basically as soon as I put this up. Changed in latest commit

File: code/game/turfs/simulated/planet.dm [hunk 2]
```diff
@@ -27,21 +30,26 @@
...
 
 /turf/open/floor/planet/dirt
+	name = "dirt"
 
 /turf/open/floor/planet/greenerdirt
 	icon_state = "greenerdirt"
 
 /turf/open/floor/planet/wood
+	gender = NEUTER
+	name = "wooden floor"
```
Comment (SamHPurp) [inline code/game/turfs/simulated/planet.dm]: hmm `wooden floor` or `wooden flooring`?

Comment (Citinited) [inline code/game/turfs/simulated/planet.dm]: I think `wooden flooring` would refer to the floor materials themselves, as opposed to the floor, which would just be `wooden floor`. At least that's how I see the difference there.

**Comment by SamHPurp** [comment]
what could go wrong

**Pull request merged**